plugins/utils/config/netconfigutils.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package config
+
+import (
+	"github.com/coreos/go-iptables/iptables"
+	"github.com/pkg/errors"
+	"net"
+	"strconv"
+	"strings"
+)
+
+// IsValidCIDR checks whether the input is a valid CIDR block and returns the IP protocol and validity
```

Comment:
Instead of creating a new utils package, we should add these functions in appropriate files under network/vpc package.